ui/src/app/components/modals.tsx

```diff
@@ -15,7 +15,9 @@ const styles = reactStyles({
 
   overlay: {
     backgroundColor: Color(colors.dark).alpha(0.85).toString(), padding: '1rem', display: 'flex',
-    position: 'fixed', left: 0, right: 0, top: 0, bottom: 0, overflowY: 'auto'
+    position: 'fixed', left: 0, right: 0, top: 0, bottom: 0, overflowY: 'auto',
+    // Keep z-index in sync with popups.tsx.
+    zIndex: 105
```

Comment:
@dolbeew FYI - this should fix the issue where the right-hand-side help bar sits above the modal overlay e.g. on saving a cohort or dataset, and was still visible/clickable. I assume this was a bug.